src/Discord.Net.Core/Entities/Interactions/MessageComponents/IComponentInteractionData.cs

```diff
@@ -18,12 +18,32 @@ public interface IComponentInteractionData : IDiscordInteractionData
         ComponentType Type { get; }
 
         /// <summary>
-        ///     Gets the value(s) of a <see cref="SelectMenuComponent"/> interaction response.
+        ///     Gets the value(s) of a <see cref="ComponentType.SelectMenu"/> interaction response.
         /// </summary>
         IReadOnlyCollection<string> Values { get; }
 
         /// <summary>
-        ///     Gets the value of a <see cref="TextInputComponent"/> interaction response.
+        ///     Gets the channels(s) of a <see cref="ComponentType.ChannelSelect"/> interaction response.
+        /// </summary>
+        IReadOnlyCollection<IChannel> Channels { get; }
```

Comment:
Are these collections `null` by default? If so, please update the xml to remark this.

Reply:
should be solved now
~~*but sure if my english.exe functions properly rn tho, didn't sleep for quite a while*~~